docker/sysbox/genrules/BUILD

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 EngFlow Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The following genrules are able to produce thousands of targets based
+# on the user's input.
+# This can be used for stress testing your build infrastructure.
+# Change the upper bound of the range and enjoy!
+UPPER_BOUND = 10
+
+[genrule(
```

Comment:
This target does not really test anything unique about Sysbox.

Reply:
modified target to use sh_rule with a simple script that runs a command inside an image